website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -11,9 +11,9 @@ We're on React Native core version 0.62 with new community modules, native modul
 
 <!--truncate-->
 
-## Upgraded to 0.62
+## Version 0.62-preview Released
 
-As promised in our previous milestone post, we're chugging along and keeping as up-to-date with the latest React Native core version as we can. That means for this latest milestone release we are on version 0.62 of React Native.
+As promised in our previous milestone post, we're chugging along and keeping as up-to-date with the latest React Native core version as we can. That means for this latest milestone release we are officially announcing our [0.62-preview of React Native for Windows](https://www.npmjs.com/package/react-native-windows/v/0.62.0-preview.6)!

```

Comment:
Recommend change to say - "As promised in our previous milestone post, we're chugging along and keeping as up-to-date with the latest React Native core version as we can. You can now start trying out the [0.62-preview of React Native for Windows](https://www.npmjs.com/package/react-native-windows/v/0.62.0-preview.6)!

Reply:
changed and also added/took from Nick's teams update announcement about the preview release.